Files/style.css

```
@@ -0,0 +1,244 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+html {
+  box-sizing: border-box;
+  font-size: 100%;
+}
+
+body {
+  font-family: sans-serif;
+  font-size: 0.9rem;
+  font-weight: 3;
+  position: relative;
+  background: #f5f5f5;
+  background-color: #eceef1;
+}
+
+nav {
+  margin-top: 10px;
+}
+
+.already {
+  float: right;
+  overflow: hidden;
+  margin-right: 20px;
+  font-size: 14px;
+  color: #757575;
+}
+
+.recovery-link {
+  float: right;
+  overflow: hidden;
+  margin-right: 20px;
+  text-decoration: none;
+  color: #0077c5;
+  font-size: 14px;
+}
+
+.sign-in-button {
+  margin-right: 20px;
+  float: right;
+  overflow: hidden;
+  color: #fff;
+  height: 35px;
+  width: 100px;
+  background: #0077c5;
+  border-radius: 4px;
+  border: 1px solid #4c4c4c;
+}
+
+.logo {
+  display: block;
+  width: 118px;
+  margin: 40px auto 18px;
+  padding-top: 20px;
+}
+
+.logos-under {
+  text-align: center;
+  margin: 15px auto;
+  justify-content: center;
+}
+
+.form-container {
+  background-color: #fff;
+  margin: auto;
+  border: 1px solid #c7c7c7;
+  width: 380px;
+  text-align: center;
+  color: #4c4c4c;
+  font-family: 'Nunito Sans', sans-serif;
+  padding-top: 30px;
+}
+
+.form-action {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.label-info {
+  display: inline-block;
+  width: 150px;
+  text-align: left;
+  margin-bottom: 8px;
+}
+
+#icons-container {
+  width: 100%;
+  height: 42px;
+  background-color: transparent;
+  text-align: center;
+}
```

You need to get familiar with the defaults as soon as possible.
```suggestion
#icons-container {
  height: 42px;
  text-align: center;
}
```